READMEs/datatokens-flow.md

```diff
@@ -76,7 +76,7 @@ config = Config('config.ini')
 ocean = Ocean(config)
 
 print("create wallet: begin")
-wallet = Wallet(ocean.web3, private_key=private_key)
+wallet = Wallet(ocean.web3, private_key=private_key, block_confirmations=config.block_confirmations)
```

Comment:
I'm wondering if we can keep this shorter for the READMEs. My suggestion: don't specify the argument names, and just have 1 confirmation. That is:
'''python 
wallet = Wallet(ocean.web3, private_key, 1)
'''

Do this for all the READMEs, for the sake of readability.

Reply:
I agree with shortening, but I disagree with the `1` literal.  Once the argument names are removed, the line is still relatively short when using `config.block_confirmations`. This is a more correct convention to show users because it accommodates any chain, not just the chains where 1 block confirmation is sufficient.